contracts/protocore/OriginProtocore.sol

```diff
@@ -130,4 +132,38 @@ contract OriginProtocore is MasterCopyNonUpgradable, GenesisOriginProtocore, Pro
             _targetBlockNumber
         );
     }
+
+
+    /* Public Functions */
+
+    /**
+     * @notice inForwardValidatorSet() function delegates the call to
+     *         the stored self protocore contract.
+     */
+    function inForwardValidatorSet(address _validator, uint256 _height)
+        public
+        view
+        returns (bool)
+    {
+        assert(selfProtocore != address(0));
```

Comment:
is `selfProtocore` not a constant `0x00..004d03` (accessed over an internal function for easy testing)

Reply:
The contract gets this from the genesis contract and is set in the `setup` function.
For the testing purpose, we set this variable from test double.